include/ebpf_nethooks.h

```diff
@@ -57,6 +57,22 @@ EBPF_HELPER(int, bpf_xdp_adjust_head, (xdp_md_t * ctx, int delta));
 #define bpf_xdp_adjust_head ((bpf_xdp_adjust_head_t)XDP_EXT_HELPER_FN_BASE + 1)
 #endif
 
+/**
+ * @brief Compute checksum difference for replacing parts of a network packet buffer.
+ *
+ * @param[in] from Pointer to raw buffer that will be replaced.
+ * @param[in] from_size Length of the "from" buffer. Must be multiples of 4.
+ * @param[in] to Pointer to the raw buffer that will replace the previous buffer.
+ * @param[in] to_size Length of the "to" buffer. Must be multiples of 4.
+ *
+ * @retval The checksum delta on success.
+ * @retval <0 A failure occured.
```

Comment:
```suggestion
 * @returns The checksum delta on success, or <0 on failure.
```
retval generates a table with the first token in the value column, and "The" is not a value.